code/index.html

```diff
@@ -4,19 +4,294 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, minimum-scale=1"
+    />
+    <script
+      src="https://kit.fontawesome.com/7ed1b41d25.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+    <div class="center-screen">
+      <a name="top"></a>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
+      <!-- ---------- HEADER--------------->
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
+      <header class="header">
+        <a class="a-logotype" href="#top">
+          <div class="logotype">
+            <span class="bold">The </span><span class="thin">Newspaper</span>
+          </div>
+        </a>
+        <a href="#">
+          <div class="icon-and-text">
+            <i class="fas fa-dollar-sign icon"></i>
+            <span>Economy</span>
+          </div>
+        </a>
+        <a href="#">
+          <div class="icon-and-text">
+            <i class="fas fa-futbol icon"></i>
+            <span>Sports</span>
+          </div>
+        </a>
+        <a href="#">
+          <div class="icon-and-text">
+            <i class="fas fa-music"></i>
+            <span>Entertainment</span>
+          </div>
+        </a>
+        <a href="#daily">
+          <div class="icon-and-text">
+            <i class="fas fa-camera-retro icon"></i>
+            <span>Daily polaroid</span>
+          </div>
+        </a>
+      </header>
+
+      <!-- ---------- BIG NEWS CONTAINER --------------->
+
+      <section class="big-news-container">
+        <div class="big-news-image">
+          <img
+            class="big-news-image"
+            src="./big-news-image.jpeg"
+            alt="A photograph of three soldiers walking with guns"
+          />
+        </div>
+        <div class="big-news-text-container">
+          <div class="big-news-text">
+            <a href="#">
+              <h3>
```

Comment:
Suggestion: use h1 and h2 before h3, especially for your biggest news. I believe it is more understandable for the search engines.

Reply:
Haha, very true! =) Did miss this =D